types/components/http/Request.d.ts

```diff
@@ -146,7 +146,7 @@ export default class Request {
 
     /**
      * Returns request headers from incoming request.
-     * @returns {Object}
+     * @returns {Record<string, unknown>}
```

Comment:
This should be Record<string, string> as header keys/values can and will only ever be strings.

Reply:
fixed